src/mixins/Projector.ts

```diff
@@ -128,12 +143,14 @@ export function ProjectorMixin<P, T extends Constructor<WidgetBase<P>>>(base: T)
 		public projectorState: ProjectorAttachState;
 
 		private _root: Element;
+		private _async = true;
 		private _attachHandle: Handle;
+		private _attachType: AttachType;
```

Comment:
Ooops I need to remove this... I thought I needed it for `.toHtml()` but didn't.